examples/php/src/Twilio/Rest/Client.php

```diff
@@ -0,0 +1,1095 @@
+<?php
+
+/**
```

Comment:
We shouldn't be including a yoyo-generated file in here. Can this be stripped down to fit the fixture docs?

Reply:
These files are added temporarily for initial PHP project setup and test php sonar checks. DII-695 (client.php) and DII-676( all the domains php files) are in play for this sprint to get rid of these yoyo files.